spec/models/video_spec.rb

```diff
@@ -5,7 +5,7 @@
   it {should validate_presence_of(:title) }
   it {should validate_presence_of(:description) }
 
-  describe "#search_by_title" do
+  describe "search_by_title" do
```

Comment:
There should be `.` instead of `#`. RSpecs has some kind of 'rules' for this. `.` is for class methods and `#` for instance methods. Read more here -> http://betterspecs.org/#describe